native-stack/README.md

```diff
@@ -239,11 +239,16 @@ Defaults to `pop`.
 Describes heights where a sheet can rest. 
 Works only when `stackPresentation` is set to `formSheet`.
 
-Available values:
+This prop can be set either to string value:
 
-- `large` - only large detent level will be allowed
-- `medium` - only medium detent level will be allowed
-- `all` - all detent levels will be allowed
+- `large` - only large detent level will be allowed (exact detent is defined by system)
+- `medium` - only medium detent level will be allowed (exact detent is defined by system)
+- `all` - all detent levels will be allowed (exact detents are defined by system)
+
+or to an array of real numbers in range [0, 1], each of which describes detent
+as a fraction of maximum detent height.
+
+Please note that the array *must* be sorted in ascending order.
```

Comment:
Maybe we could do one of the sorting methods if array is not sorted? What do you think?

Reply:
We don't want to change the value order in the array, because we send events back based on the index of the values. If we sort & change the order, the index send in the event payload would not have any meaning.